### PR TITLE
prevent members to be included more then one time in an Event

### PR DESCRIPTION
**Pull request by HennyHaelbich**
Solves #39 
Verhindert das Teilnehmer mehrfach einem Event hinzugefügt werden können

### CODE REVIEW

**Review by saskiabusch — APPROVED** [review]
Looks good :-)

File: frontend/src/addEventPage/AddEventForm.js [hunk 1]
```diff
@@ -59,8 +65,23 @@ export default function AddEventForm() {
 
   function findUser(event) {
     event.preventDefault();
-    setMember(newMember);
+    let found = false;
```
Comment (fabianschmauder) [inline frontend/src/addEventPage/AddEventForm.js:68]: you should use members.find here

File: frontend/src/addEventPage/AddEventForm.js [hunk 2]
```diff
@@ -59,8 +65,23 @@ export default function AddEventForm() {
...
+    for (let i = 0; i < members.length; i++) {
+      if (members[i].username === newMember) {
+        found = true;
+        break;
+      }
+    }
+    if (!found) {
+      setMember(newMember);
```
Comment (fabianschmauder) [inline frontend/src/addEventPage/AddEventForm.js:76]: i think you want to add a member to the list?

File: frontend/src/addEventPage/AddEventForm.js [hunk 3]
```diff
@@ -59,8 +65,23 @@ export default function AddEventForm() {
 
   function findUser(event) {
```
Comment (fabianschmauder) [inline frontend/src/addEventPage/AddEventForm.js:66]: rename the method, because you add a user. 😉

**Pull request merged**